sample.env

```diff
@@ -9,4 +9,16 @@ VITE_TRANSITIVE_USER=
 
 # The JWT secret of the above user on the above Transitive deployment's
 # Security page
-JWT_SECRET=
\ No newline at end of file
+JWT_SECRET=
+
+# Secret used by express-session
+TR_SESSION_SECRET=SOME_STRONG_SECRET_LIKE_CH9UwdjGck4DKGMbftVbBm4r9SuZQKDP
+
+# Used at build time. For local development we recommend YOUR_HOSTNAME.local
+TR_HOST=your-domain.com
+
+# create this account in accounts.json file if it doesn't exist
+TR_USER=superadmin
+TR_PASS=some-very-secure-password
+TR_EMAIL=admin_@superrobots.com
```

Comment:
Can we change our branding to "SuperBots"? It rolls of the tongue more easily than superrobots.

Reply:
Yes! That double RR made me itchy